src/app/pages/system/CloudCredentials/cloudcredentials-form/cloudcredentials-form.component.ts

```diff
@@ -128,6 +154,11 @@ export class CloudCredentialsFormComponent {
       type: 'input',
       name: 'account-B2',
       placeholder: T('Account ID'),
+      tooltip: T('<a\
+                  href="https://www.backblaze.com/b2/cloud-storage.html"\
+                  target="_blank">Backblaze B2</a> account ID. This is\
+                  an alphanumeric string found after logging into an\
```

Comment:
"found" is odd.  "displayed" or "contained in" are better, anything that doesn't imply that it is randomly found instead of of located.